internal/service/networkcontainer/resource_network_container_migration_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func TestMigNetworkContainer_basicAWS(t *testing.T) {
 	var (
-		projectID = acc.ProjectIDExecution(t)
+		projectID = acc.ProjectIDExecution(t) // No mig.ProjectIDGlobal because network container
```

Comment:
`because network container` is quite vague, could we maybe rephrase it?

Reply:
there can be only one container per project and provider / region.   I wanted to find a balance of giving some clues explaining why we're not using the "expected" way but not being very verbose  explaining the details of how Atlas works.

Reply:
as discussed offline we'll leave in this way